ui/scripts/movie.js

```diff
@@ -94,6 +95,19 @@ export default class Movie {
         return movieInfoConteiner;
     }
 
+    createDirectorsList(names) {
+        const conteiner = document.createElement('h2');
+        conteiner.classList.add("movie-directors");
+
+        if(names !== "") {
```

Comment:
isn't `names` here an iterable? why do we compare it with a string?

Reply:
good point, fixed